src/structs.py

```diff
@@ -678,6 +678,10 @@ def __gt__(self, other: object) -> bool:
         assert isinstance(other, NSRT)
         return str(self) > str(other)
 
+    def get_sampler(self) -> NSRTSampler:
```

Comment:
is there a reason to do this instead of making the sampler public, like the other NSRT attributes? that would also make `nsrt.get_sampler()(state, rng, reordered_objs)` look less weird

Reply:
changed to @property like everywhere else